microservices/ga/workerPool/README.md

```diff
@@ -160,34 +160,37 @@ This template generates the following endpoints:
 ```
 
 The liveness and readiness endpoints provide hooks for customizing
-Kubernetes Pod life cycle events.  The metrics endpoint provides an 
+Kubernetes Pod life cycle events.  The metrics endpoint provides an
 aggregated response for any Metric interfaces defined plus the standard
-metrics for the dispatcher.  The jobs and scheduler endpoints support 
-modifying the jobs currently defined or changing the schedule of the 
+metrics for the dispatcher.  The jobs and scheduler endpoints support
+modifying the jobs currently defined or changing the schedule of the
 scheduler.
 
 # Git
-The build system requires git source code management.  If the directory you choose to generate your service is not under git control, do the following after executing your template.
+The build system requires git source code management.
```

Comment:
Preflight handles this, I think we can just mention it and kill the rest off.  The .templates directory doesn't exist anymore.

Reply:
just mention that preflight.sh does this then?